Add clean command for dgraph images and volumes

twunderlich-grapl opened the pull request:
<!-- Thank you for your contribution to Grapl! Please take some time
to fill out this short template to help us review and understand your
PR. -->

### Which issue does this PR correspond to?

<!-- Please provide a link to the GitHub Issue this pull request is
meant to address. -->

### What changes does this PR make to Grapl? Why?
As we found out today, our clean command prior to this PR didn't affect anything dgraph-related. This lets us clean dgraph docker images and volumes.

In the future we should add labels to volumes
<!-- Please describe the functional impact of these changes and the
rationale behind them. This will help us understand your approach to
solving the problem. -->

### How were these changes tested?
Ran the make changes
<!-- Please describe how these changes were tested. There should be
sufficient detail that reviewers can replicate your testing
procedure. If the procedure is a manual one that's OK, your
description will help us work with you to get it automated. -->

Commit:
Add clean command for dgraph images and volumes

As we found out today, our previous clean command didn't affect anything
dgraph-related. This lets us clean dgraph docker images and volumes.

christophermaier commented on Makefile:
```diff
@@ -565,6 +571,18 @@ clean-docker-images: ## Remove all Grapl images
 		--quiet \
 	| xargs --no-run-if-empty docker rmi --force
 
+clean-docker-volumes: ## Remove all Grapl volumes
```
This isn't _all_ our volumes (it doesn't touch the ones for the build containers, for instance), just the ones generated in the course of running `docker-compose` (judging by the label), right? If so, it might be good to reflect that in the target name and the comment (as well as the comment on `DOCKER_VOLUME_FILTER_LABEL`, too).

twunderlich-grapl replied:
I've updated the comments, however since only the dgraph_export volume is labeled at the moment, I'm pondering whether it makes sense to drop the filter and just blow away all volumes. Thoughts?

christophermaier commented on Makefile:
```diff
@@ -565,6 +571,18 @@ clean-docker-images: ## Remove all Grapl images
+		--filter=label=$(DOCKER_VOLUME_FILTER_LABEL) \
+		--force
+
+clean-docker-dgraph: ## Remove dgraph images, containers and volumes
```
This command only looks like it removes images, not containers or volumes.

Commit:
Update comments around docker volume prune

Since only the dgraph_export volume is labeled at the moment, we
should explicitly call that out and mark a TODO. There is also now
a ticket for this in Zenhub
